lib/ansible/modules/utilities/logic/assert.py

```diff
@@ -38,6 +38,10 @@
     version_added: "2.7"
     description:
       - "The customized message used for a successful assertion"
+  quiet:
+    version_added: "2.8"
```

Comment:
You should add `type: bool` and `default: false`.

Reply:
Thanks Felix.  Done.

Reply:
Regarding the changelog fragment, I added it as a minor change, since the discussion on https://github.com/ansible/ansible/issues/27124 indicates this is a feature request.  I did reference the discussion in the fragment, however, as is more typical of bugfixes.  Let me know if you have any issues w/ this.

Reply:
`minor_changes` is fine for new features; I'm using that as well ;) I think the changelog fragment looks good!